### DISCUSSION TITLE
Lots of problems!

### DISCUSSION THREAD

**Discussion by domburgess** [iterate-ch/cyberduck#15720] [Q&A]
1. When opening Adobe After Effects projects, often the project will be very nearly opened, often getting up to 99% and then it just hangs and never opens. The only way to get around this is to bring the project into a new project, but this is a pain and interrupts my workflow. This seems to be happening more and more, to the point where I’m considering going back to using DropBox app.  

2. MD has started saving my random copies of my After Effects projects with long numbers at the end eg. filename.8811.215671.aep

3. Some files just don’t want to sync and I get the ‘red cross’ icon on the file. I try re-syncing the files/folders but it doesn’t work.

4. Connections/Drives don’t mount automatically on system restart.

MacBook Pro M3 Max
64gb ram
1TB SSD
Mac OS 14.3.1

**Comment by dkocher** [comment]
Please let me know if updating to version 4.15.6 resolves the issue with opening files in Adobe After Effects. To auto-connect on restart make sure you have selected _Preferences → General → Save Workspace_. For particular files not syncing you can always choose _Retry_ in the context menu [^1]. If the problem persists, write to support@mountainduck.io including the log file from _Preferencres → Connection → Log_.

[^1]: https://docs.mountainduck.io/mountainduck/connect/sync/#sync-error
